demo/meeting/src/providers/NavigationProvider.tsx

```diff
@@ -64,6 +67,10 @@ const NavigationProvider = ({ children }: Props) => {
     setShowNavbar(!showNavbar);
   };
 
+  const toggleMetrics = () => {
+    setShowMetrics(currentState => !currentState);
```

Comment:
so I saw this for the first time so currentState holds showMetrics I guess.

Reply:
Yeah if you pass a function to a setState call, you'll be passed the current state